templates/home/home_page.html

```diff
@@ -51,6 +51,24 @@ <h2 class="tna-heading tna-heading--l">
         </div>
 
     </div>
+
+<!--graph ql banner -->
+<div class="tna-container">
+
+    <div class="tna-column tna-column--width-1-5 tna-column--width-1-3-medium tna-column--full-small tna-column--full-tiny graph-banner-icon">
+        <img src ="/static/images/graph-ql-icon.jpg" alt="graphQL icon">
```

Comment:
Could not find  `/static/images/graph-ql-icon.jpg`

Reply:
Sorry, I had forgotten to add the file - should be there now.